src/components/header.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
Remove these files to `Footer.tsx`, `Header.tsx` 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/c53a344c-fc1c-4f22-aa2a-d45edd49860b" />

Reply:
Not fixed 
<img width="731" alt="image" src="https://github.com/user-attachments/assets/70f9d690-5251-43ae-9024-5ff1fb7f01c7" />